example/lib/main.dart

```diff
@@ -103,17 +122,43 @@ class _CountryListScreenState extends State<CountryListScreen> {
   }
 }
 
-class CountryDetailScreen extends StatelessWidget {
+class CountryDetailScreen extends StatefulWidget {
   final Country country;
 
   const CountryDetailScreen({Key? key, required this.country})
       : super(key: key);
 
+  @override
+  State<CountryDetailScreen> createState() => _CountryDetailScreenState();
+}
+
+class _CountryDetailScreenState extends State<CountryDetailScreen> {
+  static final _unityBannerPlacementId =
+      Platform.isAndroid ? 'Banner_Android' : 'Banner_iOS';
+
+  late final EasyUnityBannerAd _bannerAd;
```

Comment:
so till now, we haven't added the functionality to use banner ads from one file?

Reply:
no, not yet, reason cz I will integrate all the potential ad services and then I'd check if we actually can write single banner widget